tests/cypress/support/commands.ts

```diff
@@ -132,7 +132,10 @@ Cypress.Commands.add('addFleetGitRepo', ({ repoName, repoUrl, branch, path, path
     cy.get('.checkbox-outer-container.check').contains('Always Keep Resources').click();
   }
   if (correctDrift === 'yes') {
-    cy.get('[data-testid="GitRepo-correctDrift-checkbox"] > .checkbox-container > .checkbox-custom').click();
+    cy.get(
+      `[data-testid="GitRepo-correctDrift-checkbox"] > .checkbox-container > .checkbox-custom, 
```

Comment:
this locator is way too long and potentially weak.

Reply:
Let's discuss together to find optimal way.